src/entities/logical/randomEventGenerator.gd

```diff
@@ -4,8 +4,14 @@ signal random_event(event)
 
 var rng: RandomNumberGenerator = RandomNumberGenerator.new()
 var bias: int = 0 # "Bad Luck Protection" to increase chance of generating event with time
+var accumulatedDelta: float = 0
+const events = ["fire", "invaders", "hail"]
 
 func _process(delta: float) -> void:
+    accumulatedDelta += delta
```

Comment:
does this have to be a flaot? idk how it works exactly so "yes" is a completely valid answer

Reply:
yuup, delta is a float because godot said so